packages/config/config/devices/0x0086/zw075.json

```
@@ -0,0 +1,149 @@
+// AEON Labs ZW075
+// Smart Switch Gen5
+{
+	"manufacturer": "AEON Labs",
+	"manufacturerId": "0x0086",
+	"label": "ZW075",
+	"description": "Smart Switch Gen5",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0x004b",
+			"zwaveAllianceId": 1045
+		},
+		{
+			"productType": "0x0103",
+			"productId": "0x004b",
+			"zwaveAllianceId": 1053
+		},
+		{
+			"productType": "0x0203",
+			"productId": "0x004b",
+			"zwaveAllianceId": 1056
+		},
+		{
+			"productType": "0x001b",
+			"productId": "0x0048",
+			"zwaveAllianceId": 1416
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Group 2",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"2[0xff]": {
+			"$import": "templates/aeotec_template.json#blink_duration"
+		},
+		"2[0xff00]": {
+			"$import": "templates/aeotec_template.json#blink_length"
+		},
+		"3": {
+			"$import": "templates/aeotec_template.json#current_overload"
+		},
+		"20": {
+			"$if": "firmwareVersion >= 3.255",
```

Was this supposed to be
```suggestion
			"$if": "firmwareVersion >= 4.0",
```
or are they actually introducing this in version 3.255?